src/twisted/test/test_process.py

```diff
@@ -2340,6 +2340,64 @@ def test_findShebang(self):
         self.assertEqual(program, "/usr/bin/python")
 
 
+
+class Win32CreateProcessFlagsTests(unittest.TestCase):
+    """
+    Check the flags passed to CreateProcess.
+    """
+
+    @defer.inlineCallbacks
+    def test_flags(self):
+        """
+        Verify that the flags passed to win32process.CreateProcess() prevent a
+        new console window from being created. See http://tm.tl/5726
```

Comment:
I think that including the script inline (minimized as much as possible) would be better than linking to the closed ticket.